examples/prostate/pt/learners/supervised_prostate_learner.py

```diff
@@ -37,22 +37,24 @@
     RandShiftIntensityd,
     Spacingd,
 )
-from pt.learners.supervised_learner import SupervisedLearner
 from pt.utils.custom_client_datalist_json_path import custom_client_datalist_json_path
 
 from nvflare.apis.fl_context import FLContext
 from nvflare.apis.signal import Signal
 from nvflare.app_common.app_constant import AppConstants
+from nvflare.app_common.pt.learners.supervised_learner import SupervisedLearner
 from nvflare.app_common.pt.pt_fedproxloss import PTFedProxLoss
 
 
-class ProstateLearner(SupervisedLearner):
+class SupervisedProstateLearner(SupervisedLearner):
     def __init__(
         self,
+        *args,
         train_config_filename,
         aggregation_epochs: int = 1,
         train_task_name: str = AppConstants.TASK_TRAIN,
         submit_model_task_name: str = AppConstants.TASK_SUBMIT_MODEL,
+        **kwargs,
```

Comment:
Do we need *args, **kwargs here?

Reply:
yes, these are needed for multi-inheritance (same for the other class)